### PR TITLE
update readme - run tests in docker

### PR DESCRIPTION
**Pull request by ron-huberfeld**
#175 - update readme - run tests in docker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,9 +88,16 @@ After logging in, use [localhost admin](https://127.0.0.1:8080/admin) to modify
 flake8 lms
 ```
 ### Run tests
+Running tests inside the docker container:
+```
```
Comment (yammesicka) [inline README.md:92]: Why would you want to run the tests inside the docker? :)

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ron-huberfeld) [reply README.md:92]: Since all dependencies installation already done there.
Do you see any reason why not?

Comment (yammesicka) [reply README.md:92]: You can run it on your own fs, and the development shouldn't happen inside the docker anyway

Comment (ron-huberfeld) [reply README.md:92]: What do you think about this approach?
https://www.docker.com/blog/containerizing-test-tooling-creating-your-dockerfile-and-makefile/

Comment (ron-huberfeld) [reply README.md:92]: I removed this section as part of #179

**Pull request closed without merging**